test/module/libs/converter/CMakeLists.txt

```diff
@@ -1 +1,3 @@
 addtest(string_converter_test string_converter_test.cpp)
+
+target_link_libraries(string_converter_test boost)
```

Comment:
This test is also done for `common` library, so it should be linked to `common` instead of `boost` directly. Also consider the style guide mentioned above.